jib-core/src/test/java/com/google/cloud/tools/jib/image/ImageTest.java

```diff
@@ -76,4 +77,23 @@ public void test_smokeTest() throws LayerPropertyNotFoundException {
         image.getVolumes());
     Assert.assertEquals("john", image.getUser());
   }
+
+  @Test
+  public void testDefaults() {
+    Image<Layer> image = Image.builder(V22ManifestTemplate.class).build();
+    Assert.assertEquals("amd64", image.getArchitecture());
+    Assert.assertEquals("linux", image.getOs());
+    Assert.assertEquals(Collections.emptyList(), image.getLayers());
+    Assert.assertEquals(Collections.emptyList(), image.getHistory());
+  }
+
+  @Test
+  public void testOsArch() {
+    Image<Layer> image =
+        Image.builder(V22ManifestTemplate.class).setArchitecture("wasm").setOs("js").build();
+    Assert.assertEquals("wasm", image.getArchitecture());
+    Assert.assertEquals("js", image.getOs());
+    Assert.assertEquals(Collections.emptyList(), image.getLayers());
```

Comment:
What's the reason for testing the layers/history size in the os/arch test?

Reply:
Oops.  I originally had this test the must-set items.